eng/pipelines/fluent_java_mgmt_docs.yml

```diff
@@ -0,0 +1,44 @@
+trigger: none #this is not a CI, the pipeline will be triggered manually
+
+variables:
+- group: 'GitHub Release Variables'
+- name: 'GH_TOKEN'
+  value: "$(Github-azuresdkci-personalaccesstoken)"
+ 
+
+jobs:
+  - job: 'Publish_API_docs_for_the_fluent_management_java_SDK'
+  
+    steps:
+      - task: Npm@1
+        displayName: 'npm install at root'
+        inputs:
+          command: 'install'
+          
+      - script: 'chmod 775 eng/pipelines/scripts/repo'
+        displayName: 'change exec permissions on repo script'
+        
+      - script: 'cp  eng/pipelines/utils/commit.js node_modules/gift/lib/commit.js'
+        displayName: 'copy over fixed commit.js file'
+        
+      - script: 'git config --global user.email "azuresdkci@users.noreply.github.com";git config --global user.name "azuresdkci"'
```

Comment:
I'd suggest splitting this into multiple lines as it will make it easier to read.